clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookMessageHandlerTest.spec.ts

```diff
@@ -31,14 +31,15 @@ describe("Test the copybook message handler", () => {
     SettingsService.getCopybookLocalPath = jest
       .fn()
       .mockReturnValue("/configured/path/from/setting");
-    (searchCopybookInWorkspace as any) = jest
+    (searchCopybookInExtensionFolder as any) = jest
       .fn()
       .mockReturnValue("copybook content");
     expect(
       await resolveCopybookHandler(
         "cobolFileName",
         "copybookName",
         "dialectType",
+        "/storagePath",
```

Comment:
The argument is at a wrong position, which also suggest the test does not actually verify anything.

Reply:
changed : 388196d89add26b847152c098f9cc879949bc1ce

again. the whole test file does not verify anything.